README.md

```diff
@@ -1,16 +1,96 @@
-# Title
+# Git Branching
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Explain the primary role of branching
+- Explain creating a branch
+- Explain adding and committing changes
+- Explain pushing branches
+- Identify local and remote branches
+- Explain pulling branches
+- Explain merging branches
+- Explain deleting branches
 
 ## Introduction
 
-## SWBAT 1
+We're faced with many choices in life; some of these choices are easier, more convenient, or "safer" than others: Move away from familiarity or experiment with moving away and residing in a brand new city? Choose the usual ramen dish, or try it extra spicy? Style your hair the same or try cutting it all off into a radically new look?
```

Comment:
All of these are good examples to use to describe the concept of branching. But I think there might be too many of them at once. It's easier to follow if the more detailed example is introduced in the opening paragraph, or we begin right away with the upgrade example. I would also connect the upgrade example more directly with how we work with code branches.